docker-compose.yml

```diff
@@ -16,18 +16,18 @@ services:
     # substitution string. Then, the resulting subsitution from the template
     # file is copied to the real nginx.conf file before the normal nginx start
     # command is given.
-    command: /bin/bash -c "sed s@NGINX_PORT@$NGINX_PORT@ /etc/nginx/nginx-template.conf > /etc/nginx/nginx.conf && nginx -g 'daemon off;'"
+    command: /bin/bash -c "sed s@NGINX_PORT@${NGINX_PORT:-3000}@ /etc/nginx/nginx-template.conf > /etc/nginx/nginx.conf && nginx -g 'daemon off;'"
```

Comment:
🙇  This is really handy. It does seem more appropriate to be in the nginx Dockerfile instead of polluting docker-compose.yml with such a specific concern.